docs/docs/concepts/blueprints.md

```diff
@@ -1,61 +1,153 @@
 ---
-title: Blueprints
 sidebar_position: 1
 ---
 
-The data modelling tool is a tool that can be used for modelling complex domain models.
+# Blueprints and entities
 
-## What is a Blueprint?
+Blueprints can be used to create domain models. 
+
+:::info
+
+A blueprint is a model and an instance of a blueprint is called an entity. 
 
-A domain model is called for blueprint.
+:::
 
-An instances of a blueprint is called entitiy.
+## What is a Blueprint?
+
+A blueprint is described in [JSON](https://www.json.org/json-en.html). 
 
-Below is a short version of a blueprint for a Car.
+It is an JSON object that maps keys to values.
```

Comment:
This just explains a JSON object, so I suggest removing it
```suggestion
```

Reply:
Agree, think it is not needed. I removed it.